Content.Client/AME/AMEShieldingVisualizerSystem.cs

```diff
@@ -0,0 +1,65 @@
+using Content.Client.AME.Components;
+using Robust.Client.GameObjects;
+using static Content.Shared.AME.SharedAMEShieldComponent;
+
+namespace Content.Client.AME;
+
+public sealed class AMEShieldingVisualizerSystem : VisualizerSystem<AMEShieldingVisualsComponent>
+{
+    public override void Initialize()
+    {
+        base.Initialize();
+
+        SubscribeLocalEvent<AMEShieldingVisualsComponent, ComponentInit>(OnComponentInit);
+    }
+
+    private void OnComponentInit(EntityUid uid, AMEShieldingVisualsComponent component, ComponentInit args)
+    {
+        if(EntityManager.TryGetComponent<SpriteComponent>(uid, out var sprite)) {
+            sprite.LayerMapSet(AMEShieldingVisualsLayer.Core, sprite.AddLayerState("core"));
+            sprite.LayerSetVisible(AMEShieldingVisualsLayer.Core, false);
+            sprite.LayerMapSet(AMEShieldingVisualsLayer.CoreState, sprite.AddLayerState("core_weak"));
+            sprite.LayerSetVisible(AMEShieldingVisualsLayer.CoreState, false);
+        }
+    }
+
+    protected override void OnAppearanceChange(EntityUid uid, AMEShieldingVisualsComponent component, ref AppearanceChangeEvent args)
+    {
+        if(args.Sprite == null) {
+            return;
+        }
```

Comment:
```suggestion
        if(args.Sprite == null)
            return;
```

1-line ifs can be bracketless in our codebase